### ISSUE TITLE
Getting confused with syntax and versions

### ISSUE THREAD

**Issue by n0valis** [SR-G/sleep-on-lan#31]
I have downloaded the 1.0.6 snapshot version here a couple of weeks ago.
But now I don't understand this:
sol.json:

```
{
  "Listeners" : ["UDP:9", "UDP:7", "HTTP:8009" ],
  "LogLevel" : "INFO",
  "AvoidDualUDPSending" : {
	  "Active": false,
	  "Delay": "100ms"
  }
  "Commands" : [ 
    {
        "Operation" : "hibernate",
        "Command" : "nircmd hibernate",
        "Default" : "true"
    },
    {
        "Operation" : "sleep",
        "Command" : "nircmd sleep",
        "Default" : "false"
    },
    {
        "Operation" : "reboot",
        "Command" : "nircmd exitwin reboot",
        "Default" : "false"
    }]
}
```

works fine with what presumably is the 1.0.6 Version (11MB filesize, date Oct  17)
If I use it on 1.0.5 only the sleep command would be functional. For the rest I get "not found" errors.

Any explanation for that?
I can't find any syntax issues.

I have to add: If I leave
```
"AvoidDualUDPSending" : {
	  "Active": false,
	  "Delay": "100ms"
  }
```
out, it works with 1.0.5
But this section is part of the 1.0.5 sol.json

**Comment by SR-G** [comment]
Hello,
Have no idea (and was not aware) why exactly the 1.0.6 was not available anymore (indeed when being not logged, i was also not seeing it). I removed the "pre-release" flags and now it should again be visible ... (but older versions also have that flag ...).
https://github.com/SR-G/sleep-on-lan/releases/tag/1.0.6-SNAPSHOT

I have no idea right now why there would be a difference about the "commands" definition inside the configuration (at first glance, it looks fine with your example). I would suggest to use 1.0.6 (available again).

**Comment by n0valis** [comment]
Yes I changed my post after I discovered it. 1.0.6 Was not visible but after I logged in and went to releases I found it.
Nevertheless AvoidDualUDPSending seems to be the issue. It doesn't work with 1.0.5 but is present inside the sol.json shipped with 1.0.5

**Comment by SR-G** [comment]
Correct, this is a new field only in 1.0.6 (i thought the problem was with commands).
I'll give a look about behavior in case of configuration errors, just in case i can make that more understandable.

**Comment by n0valis** [comment]
Well it is both then, if AvoidDualUDPSending is used with 1.0.5 (which the sol.json suggests) then the commands don't work properly. In fact only the sleep command is functional.

**Comment by SR-G** [comment]
Yes, i fully understand now : because of the extra (unknown) configuration field while running 1.0.5, the loading configuration was failing - after that, the program was still pursuing its execution, but with default values (sleep command is configured as a default one). Hence the behavior you spotted.

I'll get it corrected (by stopping the program in case of any configuration errors, it's way safer / nicer this way).

**Comment by SR-G** [comment]
Closing as this is now corrected through #32 + 1.0.7-SNAPSHOT
